docker-compose.yml

```diff
@@ -53,11 +53,24 @@ services:
     network_mode: ${NETWORK_MODE}
     container_name: ${CONTAINER_NAME_PREFIX}powersupply
     depends_on:
-      - databaseds
+      - dsconfig
     environment:
       - TANGO_HOST=${TANGO_HOST}
     command: >
       sh -c "wait-for-it.sh ${TANGO_HOST} --timeout=30 --strict --
-             tango_admin --check-device test/power_supply/1 ||\
-             tango_admin --add-server PowerSupply/test PowerSupply test/power_supply/1 &&\
              /venv/bin/python /app/powersupply/powersupply.py test"
+
+  dsconfig:
+    image: nexus.engageska-portugal.pt/ska-docker/tango-dsconfig:latest
+    network_mode: ${NETWORK_MODE}
+    container_name: ${CONTAINER_NAME_PREFIX}dsconfig
+    depends_on:
+      - databaseds
+    environment:
+      - TANGO_HOST=${TANGO_HOST}
+    command: >
+      sh -c "wait-for-it.sh ${TANGO_HOST} --timeout=30 --strict --
+             json2tango -w tango_example/powersupply/config.json &&\
+              sleep 3m"
```

Comment:
@Matteo04052017 I'm still trying to see if there is a way to keep this service running after it has registered the device to the database.

Reply:
well, that's another story. I know that at max4 they added a grpc interface for it.